doc/api/next_api_changes/2018-09-19-AL.rst

```diff
@@ -0,0 +1,9 @@
+`matplotlib.rc_context` is now a `contextlib.contextmanager`
```

Comment:
I think this should go in "What's new" instead of API changes.

Reply:
sure, moved